nengo_sphinx_theme/theme/versions.html

```diff
@@ -8,20 +8,20 @@
         <nav class="w-dropdown-list dropdown-list">
             {% set prefix = pagename | list | select("equalto", "/") | map("replace", "/", "../") | join("") %}
             {% if building_version == "latest" %}
-                <div class="w-dropdown-link">latest</div>
+            <div class="w-dropdown-link">latest</div>
```

Comment:
If we're taking a vote I prefer having the template well indented, as I spend a lot more time looking at that then the rendered HTML.  But I don't feel super strongly about it.